package.json

```diff
@@ -4,8 +4,10 @@
   "version": "0.3.7",
   "license": "MIT",
   "dependencies": {
+    "abortcontroller-polyfill": "^1.4.0",
```

Comment:
On IE11, I still get the error message

`ReferenceError: "AbortController" ist undefiniert`

althought the deploy preview has been updated on https://app.netlify.com/sites/react-spatial/deploys.

But, in GitLab, I still see an old timestamp for the deploy preview.

Currently Production/master is published. If I click on the publish button in https://app.netlify.com/sites/react-spatial/deploys/5eec94b138650a7d81406c94, this would be overwritten, or?

@oterral, any idea?

Reply:
> Currently Production/master is published. If I click on the publish button in https://app.netlify.com/sites/react-spatial/deploys/5eec94b138650a7d81406c94, this would be overwritten, or?

yes

Reply:
> On IE11, I still get the error messag e ReferenceError: "AbortController" ist undefinier

See react-transit config https://github.com/geops/react-transit/blob/master/styleguide.config.js#L17

Reply:
No, it works. Thanks!